Samples/3b-custom-transformations/generated/pipeline.yaml

```diff
@@ -51,20 +51,18 @@ swagger-document/individual/azure-validator:
     - azure-validator-individual
   inputs:
     - swagger-document/individual/transform
-swagger-document/azure-openapi-validator:
-  pluginName: azure-openapi-validator
+swagger-document/azure-openapi-linter:
+  pluginName: azure-openapi-linter
   configScope:
     - azure-validator-composed
   inputs:
     - swagger-document/transform
-    - swagger-document/azure-validator
-swagger-document/individual/azure-openapi-validator:
-  pluginName: azure-openapi-validator
+swagger-document/individual/azure-openapi-linter:
+  pluginName: azure-openapi-linter
   configScope:
     - azure-validator-individual
   inputs:
     - swagger-document/individual/transform
-    - swagger-document/individual/azure-validator
```

Comment:
`swagger-document/individual/azure-validator` scope used to set the `merge state` to `composed` only. By assigning proper configScopes to both `swagger-document/azure-openapi-linter` and `swagger-document/individual/azure-openapi-linter` we should be able to run the linter in both `composed` and `individual` merge states.